src/main/java/world/bentobox/bentobox/managers/AddonsManager.java

```diff
@@ -176,6 +179,7 @@ private void handleAddonError(@NonNull Addon addon, @NonNull Throwable throwable
         addon.setState(Addon.State.ERROR);
         plugin.logError("Skipping " + addon.getDescription().getName() + " due to an unhandled exception...");
         plugin.logError("STACKTRACE: " + throwable.getClass().getSimpleName() + " - " + throwable.getMessage() + " - " + throwable.getCause());
+        throwable.printStackTrace();
         if (plugin.getConfig().getBoolean("debug")) {
```

Comment:
May have to remove the "debug" hidden thing then?

Reply:
Yes. I spotted another issue too. Let me fix those.